packages/esm-patient-search-app/src/compact-patient-search/compact-patient-banner.component.tsx

```diff
@@ -123,13 +123,7 @@ const CompactPatientBanner = forwardRef<HTMLDivElement, CompactPatientBannerProp
                 {getGender(patient.gender)} <span className={styles.middot}>&middot;</span> {age(patient.birthDate)}
                 <span className={styles.middot}>&middot;</span>
                 {config.defaultIdentifierTypes.length ? (
-                  <>
-                    {patientIdentifiers.length > 1 ? (
-                      <DefaultIdentifiers identifiers={patientIdentifiers} />
-                    ) : (
-                      <FallbackIdentifier patient={patients[index]} identifierName={config.defaultIdentifier} />
-                    )}
-                  </>
+                  <DefaultIdentifiers identifiers={patientIdentifiers} />
```

Comment:
What happens if the configuration still specifies no patient identifiers? It seems like we still need some fallback here.

Reply:
@ibacher how do you suggest we handle this callback, given that from the previous implementations it was checking for a static string `OpenMRS ID` if it existed in the patient identifiers

```typescript
const FallbackIdentifier: React.FC<CustomIdentifierProps> = ({ patient, identifierName }) => {
  const identifier = patient.identifiers.find((identifier) => identifier.identifierType.display === identifierName);

  return identifier ? <IdentifierTag identifier={identifier} /> : null;
};
```

Reply:
@usamaidrsk Technically, it's checking for a user-configurable value, `identifierName`, right? I'm not sure there's a saner implementation than that. Maybe if that's also unset we could look for the `preferred` identifier?

Reply:
> Technically, it's checking for a user-configurable value, identifierName, right?.

Yes it is a configurable value, which makes it not safe.

I do agree with the idea of the `preferred` identifier, how do we get to it
cc @ibacher

Reply:
It's a property of `patient.identifiers` or available as one (if it's excluded via a custom representation).